source/_patterns/02-molecules/vue-widgets/src/vue-cards/app.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div id="vue-cards">
+    <h2 id="mt-3'">Vue Styles</h2>
```

Comment:
What is `id="mt-3"`? Do you mean the Bootstrap `class="mt-3"` for 3x the margin top style?